src/main/java/core/basesyntax/service/impl/ShiftReportServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ShiftReportService;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ShiftReportServiceImpl implements ShiftReportService {
+    private int counter;
```

Comment:
remove this variable and rewrite your logic accordingly

Reply:
OK, remove last element from List